### ISSUE TITLE
[Bug] Backbutton press throws error

### ISSUE THREAD

**Issue by ddsky** [quasarframework/quasar#2284]
### Problem ###
Back button works in browser but not in cordova app (problem new since upgrade from quasar 0.14 to 0.16).

### Software version

Operating System: Windows_NT(10.0.17134) - win32/x64
NodeJs: 8.11.3
NPM: 6.1.0
yarn: 1.7.0
quasar-cli: 0.16.4
vue-cli: 2.9.6
cordova: 8.0.0

### What did you get as the error?
```
Uncaught TypeError: Cannot read property 'style' of undefined
    at u (vendor.026357d5.js:formatted:1383)
    at s.applyPosition (vendor.026357d5.js:formatted:9832)
    at s.__hide (vendor.026357d5.js:formatted:9903)
    at s.hide (vendor.026357d5.js:formatted:2045)
    at Channel.<anonymous> (vendor.026357d5.js:formatted:1514)
    at Channel.fire (cordova.js:842)
    at cordova.js:229
```

### What were you expecting?
No error, working back button.

### What steps did you take, to get the error?
I run `quasar build -m cordova -T android` from root folder and then `cordova run android --device` from `src-cordova` folder.

On device I open a route and press the android back button.

**Comment by rstoenescu** [comment]
Hi,

Unfortunately, the description is not very helpful. Can you tell what exactly from your vendor.js is generating the error? Or at least offer a minimal reproduction repo. Otherwise I can't investigate. Thank you.

**Comment by rstoenescu** [comment]
Investigated directly on ddsky's repo and fixed corner-case issue. Available in future v0.17 release.
